src/pages/RecipeNew.js

```diff
@@ -1,8 +1,89 @@
-import React from 'react'
+import React, { useState } from 'react'
+import { useNavigate } from 'react-router-dom'
+import { Button, Form, FormGroup, Input, Label } from 'reactstrap'
+
+const RecipeNew = ({createRecipe}) => {
+
+  const navigate = useNavigate()
+
+  const [newRecipe, setNewRecipe] = useState ({
+    name: "", 
+    instructions: "",
+    cook_time: "",
+    ingredients: "",
+    course: "",
+    servings: "",
+    allergies: "",
+    tutorial_link: "",
+    recipe_image: "",
+  })
+
+  const handleChange = (e) => {
+    setNewRecipe({ ...newRecipe, [e.target.name]: e.target.value})
+  } 
+
+  const handleSubmit = () => {
+    createRecipe(createRecipe)
```

Comment:
Are you passing the function of createRecipe to createRecipe? What kind of response are you getting from this?